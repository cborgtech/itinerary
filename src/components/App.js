import React, { Component } from "react";
import "./App.css";
import { ListGroup, ListGroupItem } from "react-bootstrap";
import Menu from "./Menu";

class App extends Component {
  render() {
    return (
      <div className="App">
        <Menu />
        <ListGroup>
          <ListGroupItem header="Monday" href="#">
            February 1, 2018
          </ListGroupItem>
          <ListGroupItem header="Tuesday" href="#">
            February 1, 2018
          </ListGroupItem>
          <ListGroupItem header="Wednesday" href="#">
            February 1, 2018
          </ListGroupItem>
        </ListGroup>
        <p className="footer">Courtesy of CBORGTECH</p>
      </div>
    );
  }
}

export default App;
