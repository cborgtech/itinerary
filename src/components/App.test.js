import React from "react";
// import ReactDOM from "react-dom";
import "../setupTest.js";
import { shallow } from "enzyme";
import App from "./App";

// it("renders without crashing", () => {
//   const div = document.createElement("div");
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });

describe("App component", () => {
  const wrapper = shallow(<App />);
  it("renders without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });

  //it has a div and the div has 2 children

  // it("contains a Menu component", () => {
  //   expect(wrapper.find("Menu").exists()).toBe(true);
  // });
});
