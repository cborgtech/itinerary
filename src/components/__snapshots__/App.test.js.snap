// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(Menu) />,
        <ListGroup
          bsClass="list-group"
>
          <ListGroupItem
                    bsClass="list-group-item"
                    header="Monday"
                    href="#"
                    listItem={false}
          >
                    February 1, 2018
          </ListGroupItem>
          <ListGroupItem
                    bsClass="list-group-item"
                    header="Tuesday"
                    href="#"
                    listItem={false}
          >
                    February 1, 2018
          </ListGroupItem>
          <ListGroupItem
                    bsClass="list-group-item"
                    header="Wednesday"
                    href="#"
                    listItem={false}
          >
                    February 1, 2018
          </ListGroupItem>
</ListGroup>,
        <p
          className="footer"
>
          Courtesy of CBORGTECH
</p>,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "bsClass": "list-group",
          "children": Array [
            <ListGroupItem
              bsClass="list-group-item"
              header="Monday"
              href="#"
              listItem={false}
>
              February 1, 2018
</ListGroupItem>,
            <ListGroupItem
              bsClass="list-group-item"
              header="Tuesday"
              href="#"
              listItem={false}
>
              February 1, 2018
</ListGroupItem>,
            <ListGroupItem
              bsClass="list-group-item"
              header="Wednesday"
              href="#"
              listItem={false}
>
              February 1, 2018
</ListGroupItem>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "list-group-item",
              "children": "February 1, 2018",
              "header": "Monday",
              "href": "#",
              "listItem": false,
            },
            "ref": null,
            "rendered": "February 1, 2018",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "list-group-item",
              "children": "February 1, 2018",
              "header": "Tuesday",
              "href": "#",
              "listItem": false,
            },
            "ref": null,
            "rendered": "February 1, 2018",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "bsClass": "list-group-item",
              "children": "February 1, 2018",
              "header": "Wednesday",
              "href": "#",
              "listItem": false,
            },
            "ref": null,
            "rendered": "February 1, 2018",
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Courtesy of CBORGTECH",
          "className": "footer",
        },
        "ref": null,
        "rendered": "Courtesy of CBORGTECH",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(Menu) />,
          <ListGroup
            bsClass="list-group"
>
            <ListGroupItem
                        bsClass="list-group-item"
                        header="Monday"
                        href="#"
                        listItem={false}
            >
                        February 1, 2018
            </ListGroupItem>
            <ListGroupItem
                        bsClass="list-group-item"
                        header="Tuesday"
                        href="#"
                        listItem={false}
            >
                        February 1, 2018
            </ListGroupItem>
            <ListGroupItem
                        bsClass="list-group-item"
                        header="Wednesday"
                        href="#"
                        listItem={false}
            >
                        February 1, 2018
            </ListGroupItem>
</ListGroup>,
          <p
            className="footer"
>
            Courtesy of CBORGTECH
</p>,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "bsClass": "list-group",
            "children": Array [
              <ListGroupItem
                bsClass="list-group-item"
                header="Monday"
                href="#"
                listItem={false}
>
                February 1, 2018
</ListGroupItem>,
              <ListGroupItem
                bsClass="list-group-item"
                header="Tuesday"
                href="#"
                listItem={false}
>
                February 1, 2018
</ListGroupItem>,
              <ListGroupItem
                bsClass="list-group-item"
                header="Wednesday"
                href="#"
                listItem={false}
>
                February 1, 2018
</ListGroupItem>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "list-group-item",
                "children": "February 1, 2018",
                "header": "Monday",
                "href": "#",
                "listItem": false,
              },
              "ref": null,
              "rendered": "February 1, 2018",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "list-group-item",
                "children": "February 1, 2018",
                "header": "Tuesday",
                "href": "#",
                "listItem": false,
              },
              "ref": null,
              "rendered": "February 1, 2018",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "bsClass": "list-group-item",
                "children": "February 1, 2018",
                "header": "Wednesday",
                "href": "#",
                "listItem": false,
              },
              "ref": null,
              "rendered": "February 1, 2018",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Courtesy of CBORGTECH",
            "className": "footer",
          },
          "ref": null,
          "rendered": "Courtesy of CBORGTECH",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
